src/extensions/default/Brackets-InteractiveLinter/main.js

```diff
@@ -21,6 +21,18 @@ define(function (require, exports, module) {
         linterManager      = require("linterManager"),
         pluginManager      = require("pluginManager");
 
+        
+        var defaultPreferences = { 
```

Comment:
This entire code block (line 25-34) is indented one tab stop too far.  Make it line up with what's above and below it